### PR TITLE
Added products page to the ecommerce site (includes code clean up and refactoring )

### PR DESCRIPTION
**Pull request by sunilvr**

### CODE REVIEW

**Comment by sunilvr** [comment]
image

image

**Comment by sunilvr** [comment]
image

image

**Review by Vishnu-Aithal — APPROVED** [review]
Looks good to merge. 
edit:
Also mikesch803 has pointed out some critical changes required.

File: products.html [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous" />
+    <script src="https://kit.fontawesome.com/5085ec57f6.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="flex-column">
+        <header class="header flex-row">
+            <div class="logo-title">
+                <button class="btn-n-border">
```
Comment (mikesch803) [inline products.html]: why you have wrapped your logo into btn

File: products.html [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+
+        <footer class="footer flex-column">
+            <div class="footer-content flex-row ">
+                <div class="footer-column flex-column">
+                    <span class="footer-headings txt-bold p-lg">Get to know us</span>
+                    <button class="primary-btn-link p-md txt-bold">Contact</button>
+                    <button class="primary-btn-link p-md txt-bold">About Us</button>
+                    <button class="primary-btn-link p-md txt-bold">Careers</button>
+                    <button class="primary-btn-link p-md txt-bold">Press Releases</button>
+                    <button class="primary-btn-link p-md txt-bold">Stories</button>
```
Comment (mikesch803) [inline products.html]: you should prefer the links instead btns

File: products.html [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+                    <button class="primary-btn-link p-md txt-bold">Security</button>
+                    <button class="primary-btn-link p-md txt-bold">Privacy</button>
+                </div>
+                <div class="footer-column flex-column">
+                    <span class="footer-headings txt-bold p-lg">Social</span>
+                    <button class="primary-btn-link p-md txt-bold">Facebook</button>
+                    <button class="primary-btn-link p-md txt-bold">Instagram</button>
+                    <button class="primary-btn-link p-md txt-bold">Twitter</button>
+                    <button class="primary-btn-link p-md txt-bold">Youtube</button>
+                    <button class="primary-btn-link p-md txt-bold">LinkedIn</button>
```
Comment (mikesch803) [inline products.html]: this is link should use anchor tag
add only btn when you will be adding some functionality or interaction

File: products.html (hunk 1 above)
Comment (sunilvr) [reply products.html]: noted. Made the change Mahendra

File: products.html (hunk 2 above)
Comment (sunilvr) [reply products.html]: changed done and pushed.  Thanks for the review Mahendra

File: products.html (hunk 3 above)
Comment (sunilvr) [reply products.html]: changes done

**Comment by sunilvr** [comment]
> Looks good to merge. edit: Also mikesch803 has pointed out some critical changes required.

thanks for the review Vishnu

**Pull request merged**